### PR TITLE
feature: superAdminUserManagement, added based on role, init admin dashboard.

### PR DESCRIPTION
**Pull request by SyaibanAhmadRamadhan**
TODO: bug button apply filter svelte

### CODE REVIEW

File: domain/0_main_test.go [hunk 1]
```diff
@@ -171,8 +171,8 @@ func testDomain() (*Domain, func()) {
 		AuthOltp: testTt,
 		AuthOlap: testCh,
 
-		PropOltp: testTt,
-		PropOlap: testCh,
+		// PropOltp: testTt,
```
Comment (kokizzu) [inline domain/0_main_test.go:174]: delete juga sekalian yg di Domain

**Comment by kokizzu** [comment]
`*.fiber.gz` tolong masukkan `.gitignore`, nanti waktu merge di squash biar ga ikut tercommit

File: svelte/_components/formatter.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    if( !humanize ) {
+        dt = dt.toISOString();
+        return dt.substring( 0, 10 ) + ' ' + dt.substring( 11, 16 );
+    }
+    const options = {day: '2-digit', month: 'long', year: 'numeric'};
+    const formattedDate = dt.toLocaleDateString( undefined, options );
+    return formattedDate;
+}
+
+function priceNtd( val ) {
```
Comment (kokizzu) [inline svelte/_components/formatter.js:14]: hapus :3

File: svelte/_components/formatter.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    const year = dt.getFullYear();
+    let hh = dt.getHours();
+    if( hh<10 ) hh = '0' + hh;
+    let mm = dt.getMinutes();
+    if( mm<10 ) mm = '0' + mm;
+    const formattedDate = `${day}, ${date} ${month} ${year} ${hh}:${mm}`;
+    return formattedDate;
+}
+
+function datetime2( unixSec ) {
```
Comment (kokizzu) [inline svelte/_components/formatter.js:33]: hapus

File: svelte/_components/formatter.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    const day = dt.toLocaleDateString( 'default', {weekday: 'long'} );
+    const date = String( dt.getDate() ).padStart( 2, '0' );
+    const hours = String( dt.getHours() ).padStart( 2, '0' );
+    const minutes = String( dt.getMinutes() ).padStart( 2, '0' );
+    const seconds = String( dt.getSeconds() ).padStart( 2, '0' );
+
+    return `${day}, ${date} ${month} ${year} - ${hours}:${minutes}:${seconds}`;
+}
+
+function formatPrice( price, currency ) {
```
Comment (kokizzu) [inline svelte/_components/formatter.js:48]: hapus

File: svelte/_components/uiState.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+export let isSideMenuOpen = writable(false); // Side Menu
+
+let original = {};
+export let langOptions = {
+    en: 'EN',
+    id: 'ID', // add more languages here
+};
+
+function translationStore() {
```
Comment (kokizzu) [inline svelte/_components/uiState.js:12]: ngga usah pakai translation, ntar gabung EN dan ID langsung di UI

File: domain/0_main_test.go (hunk 1 above)
Comment (SyaibanAhmadRamadhan) [reply domain/0_main_test.go]: done

File: svelte/_components/formatter.js (hunk 3 above)
Comment (SyaibanAhmadRamadhan) [reply svelte/_components/formatter.js]: done

File: svelte/_components/formatter.js (hunk 4 above)
Comment (SyaibanAhmadRamadhan) [reply svelte/_components/formatter.js]: done

File: svelte/_components/uiState.js (hunk 5 above)
Comment (SyaibanAhmadRamadhan) [reply svelte/_components/uiState.js]: done

File: svelte/_components/formatter.js (hunk 4 above)
Comment (SyaibanAhmadRamadhan) [reply svelte/_components/formatter.js]: done

**Review by kokizzu — APPROVED** [review]
LGTM

**Comment by kokizzu** [comment]
pilih Squash and merge ya

**Pull request merged**